Cython/Utility/ObjectHandling.c

```diff
@@ -2553,3 +2553,132 @@ static PyObject *__Pyx_PyMethod_New(PyObject *func, PyObject *self, CYTHON_UNUSE
 #else
     #define __Pyx_PyMethod_New PyMethod_New
 #endif
+
+/////////////// UnicodeConcatInPlace.proto ////////////////
+
+#if CYTHON_COMPILING_IN_CPYTHON && PY_MAJOR_VERSION > 2
+    #if CYTHON_REFNANNY
+        #define __PYX_PYUNICODE_CONCATINPLACE(a, b) __Pyx_PyUnicode_ConcatInPlaceImpl(&a, b, __pyx_refnanny)
```

Comment:
All-upper-case macro names are difficult to read and also result in a different shade of yellow in `cython -a`, because they are assumed to be simpler than something that looks like a function.

Besides, I really don't see why these names should be syntactically different from the function names that they call.

Reply:
It was mostly to give anyone reading the code a clue that it might do something unexpected (i.e. modify the arguments). However, I'll switch them back.